pieceofcake/__init__.py

```diff
@@ -1 +1 @@
-from .version import version as __version__
+#from .version import version as __version__
```

Comment:
@namurphy - was this change intentional?

Reply:
Oops!  Nope; the line gave me a problem when I did a local pip install or something (I forgot exactly what) so I commented out rather than trying to figure out what was going on.